### ISSUE TITLE
set MOUSE_ELAN_I2C_SMBUS=y

### ISSUE THREAD

**Issue by sn99** [clearlinux/distribution#2252]
`psmouse serio1: elantech: The touchpad can support a better bus than the too old PS/2 protocol. Make sure MOUSE_PS2_ELANTECH_SMBUS and MOUSE_ELAN_I2C_SMBUS are enabled to get a better touchpad experience.`

**Comment by RussianE39** [comment]
Bumping up the issue, most of modern thinkpads using ELAN touchpad driver, so its definitely worth enabling it for both lts and latest kernels.

**Comment by fenrus75** [comment]
it's a loadable module... that shold be enough right?

**Comment by RussianE39** [comment]
No, its not. Depending on MOUSE_PS2_ELANTECH_SMBUS setting, elantech module built in different way.

**Comment by fenrus75** [comment]
SMBUS was NOT turned on at all.. likely that's the problem so I'm fixing
that at least

**Comment by RussianE39** [comment]
Thank you very much, finally one of the long standing issues to solve
